nlp_primitives/mean_characters_per_word.py

```diff
@@ -39,6 +40,6 @@ def mean_characters_per_word(x):
             df = df.reset_index()
             df = df.melt(id_vars='index', var_name='word_index', value_name='word')
             df['n_characters'] = df['word'].str.len()
-            results = df.groupby(by=['index']).mean()['n_characters']
+            results = df.groupby(by=['index']).mean(numeric_only=False)['n_characters']
```

Comment:
This change is required because in situations where the column of words contains `pd.NA` values the `n_characters` column has a dtype of `object` which will not have `mean` calculated by default.